### PR TITLE
add screener config environment variables

### PR DESCRIPTION
**Pull request by mikrotron**
### Purpose 🚀
_clear, one sentence description of primary purpose of this PR_

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [ ] MINOR (backward compatible) change to _these packages_
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -4,20 +4,25 @@ agent:
   machine:
     type: e1-standard-2
     os_image: ubuntu1804
-
 blocks:
-  - name: "🎩 Yarn"
+  - name: "\U0001F3A9 Yarn"
```
Comment (alexzherdev) [inline .semaphore/semaphore.yml]: huh, does this actually make a difference?

Comment (mikrotron) [reply .semaphore/semaphore.yml]: Uh... weird.  Not sure what made that change, but it was applied automatically by something...

Comment (alexzherdev) [reply .semaphore/semaphore.yml]: prettier most likely

Comment (mikrotron) [reply .semaphore/semaphore.yml]: Aha, it was Semaphore! It makes the commit when you grant access to the secret.

**Pull request closed without merging**